qp/pdf.py

```diff
@@ -122,27 +124,33 @@ def evaluate(self, loc, vb=True, using=None):
 
         return(loc, val)
 
-    def integrate(self, limits):
+    def integrate(self, limits, dx=0.0001, using=None):
         """
         Computes the integral under the PDF between the given limits.
 
         Parameters
         ----------
-        limits: float, tuple
+        limits: tuple, float
             limits of integration
+        dx: float, optional
+            granularity of integral
+        using: string, optional
```

Comment:
Maybe a note on what values "using" can use, or mention that its that same as what "evaluate" uses?